Add files via upload

DavidMorin99 opened the pull request:
Added links to MATLAB code

DavidMorin99 committed:
Add files via upload

Added links to MATLAB code

sarboleda22 commented on docs/Electric_Field_Voltage_Paper_Simulation.m:
```diff
@@ -0,0 +1,26 @@
+plot(x3,V1)
+hold on
+plot(x4,V2), xlabel('depth'), ylabel('Voltage'), title('Figure 21: Voltage v Depth'), legend('10^2^4 m^-3', '10^2^3 m^-3')
+hold off
```
Add an empty line at the end (EOF)

sarboleda22 commented on docs/Electric_Field_Voltage_Simulation.m:
```diff
@@ -0,0 +1,24 @@
+V1=((e*NA)/(2*epsilon))*(x1+wp).^2;             %for -wp<x<0
+V2=V0-(((e*ND)/(2*epsilon))*(x2-wn).^2);        %for 0<x<wn
+Vfinal=[V1,V2];                                 %vector combining V1 and V2
+figure(2), plot(xfinal,Vfinal), xlabel('x'), ylabel('V(x)'), title('Figure 16: Voltage v Position of an Abrupt P-N Junction')
```
Add new line (EOF)

DavidMorin99 committed:
Add files via upload

Added a line at the bottom of both files.
Created a simulation folder added it to documentation
Added MATLAB code to simulation folder

DavidMorin99 committed:
Delete Electric_Field_Voltage_Paper_Simulation.m

DavidMorin99 committed:
Delete Electric_Field_Voltage_Simulation.m

sarboleda22 commented on simulations/Electric_Field_Voltage_Paper_Simulation.m:
```diff
@@ -0,0 +1,27 @@
+plot(x3,V1)
+hold on
+plot(x4,V2), xlabel('depth'), ylabel('Voltage'), title('Figure 21: Voltage v Depth'), legend('10^2^4 m^-3', '10^2^3 m^-3')
+hold off
```
Add line at EOF

DavidMorin99 committed:
Update Electric_Field_Voltage_Paper_Simulation.m

Added space

sarboleda22 reviewed (approved):
:+1:
